src/integration_tests/mission_transition.cpp

```diff
@@ -0,0 +1,178 @@
+#include <iostream>
+#include <functional>
+#include <memory>
+#include <future>
+#include <atomic>
+#include <cmath>
+#include "integration_test_helper.h"
+#include "path_checker.h"
+#include "mavsdk.h"
+#include "plugins/telemetry/telemetry.h"
+#include "plugins/action/action.h"
+#include "plugins/mission/mission.h"
+
+using namespace mavsdk;
+using namespace std::placeholders; // for `_1`
+
+void do_mission_takeoff_transitions_land(float mission_altitude_m);
+
+TEST_F(SitlTest, MissionTakeoffTransitionAndLandHigh)
+{
+    do_mission_takeoff_transitions_land(40);
+}
+
+void do_mission_takeoff_transitions_land(float mission_altitude_m)
```

Comment:
Same thing as above. Why `High`? And you can just put the function into the `TEST_F`. That way the `ASSERT` can actually stop the test, once they are a function deep they can't and are like `EXPECT`.

Reply:
You mean to remove do_mission_takeoff_transitions_land and put test into TEST_F?